### DISCUSSION TITLE
How to read juicefs profile <mount-point>/.accesslog

### DISCUSSION THREAD

**Discussion by myc724** [juicedata/juicefs#706] [Q&A] [answered]
Is there any document explaining the meaning of these columns? Thanks! It seems columns #3 and #4 are the averages and total times to execute each operation. How about column #5 (where some are over 100%)? The profiling doc does not have a column ?#5.

https://github.com/juicedata/juicefs/blob/main/docs/en/operations_profiling.md

![image](https://user-images.githubusercontent.com/48360018/128384448-439ebe9d-a97b-4112-9c64-d452d2709996.png)

**Comment by xiaogaozi** [comment, marked as answer]
The "Percent(%)" column equals `Total(us) / Interval`. For example in your "example output 2", the `Total(us)` of `lookup` operation is 2517482, the default value of `Internal` is 2 seconds (specify by `--interval` option), so `2517482 / 2000000 * 100 = 125.9`.

For operation that percentage over 100%, it means there're multiple processes or multiple threads call this operation.

**Reply by myc724** [reply]
Got it. Thanks!
